rpc-control.go

```diff
@@ -26,13 +27,19 @@ import (
 const (
 	controlRPCPath = reservedBucket + "/control"
 	healPath       = controlRPCPath + "/heal"
+	shutdownPath   = controlRPCPath + "/shutdown"
```

Comment:
Check the refactor here - https://github.com/minio/minio/commit/1d9617044348d2ab556eab005f55e7f8331e3553, once your patch is merged will rebase distributed branch properly.